packages/backend-common/src/dynamodb.ts

```diff
@@ -31,19 +31,19 @@ export const Transcripts = z.object({
 
 export type Transcripts = z.infer<typeof Transcripts>;
 
-export const TranscriptionItem = z.object({
+export const TranscriptionDynamoItem = z.object({
 	id: z.string(),
 	originalFilename: z.string(),
-	transcripts: Transcripts,
+	transcriptKeys: Transcripts,
```

Comment:
Would it not be better if we change the type name to something like `TranscriptKeys`?